build/dsc/property.rb

```diff
@@ -21,6 +21,7 @@ def description
       begin
         if @description.nil? && @cim_feature.description
           content = @cim_feature.description
+          content.force_encoding(Encoding::UTF_8)
```

Comment:
I'm a little concerned about the use of `force_encoding` here as it wipes out the strings encoding. Was this a result of loading files as `BINARY`? FWIW, it's better to explicitly specify the encoding when opening the file for reading in the first place (like `'BOM|UTF-8'` - see https://ruby-doc.org/core-2.0.0/IO.html#method-c-new-label-IO+Encoding)